openmc_plotter/main_window.py

```diff
@@ -1216,3 +1211,15 @@ def saveSettings(self):
     def exportTallyData(self):
         # show export tool dialog
         self.showExportDialog()
+
+    @staticmethod
+    def hash_file(filename):
+        # return the md5 hash of a file
+        h = hashlib.md5()
+        with open(filename,'rb') as file:
+            chunk = 0
+            while chunk != b'':
+                # read 1024 bytes at a time
+                chunk = file.read(1024)
+                h.update(chunk)
+        return h.hexdigest()
```

Comment:
I would just make this a regular function instead of a staticmethod of the MainWindow class. Also, I assume the chunking is intended to avoid loading the whole file in memory -- I would make the chunk size a bit bigger, perhaps 8192 or 32768.

Reply:
Yes, using a `chunk` seemed to be the consistent method in all the examples I found on how to do this with a file. I can make it larger.